schemas/derivatives/SCHEMA_AMM (with vault)

```diff
@@ -0,0 +1,116 @@
+# Derivatives
+
+OpenBlock Labs standard schema for derivatives protocols (AMM Perps).
+
+## Version: 1.0.0-alpha
+
+### Pairs
+
+Pairs traded in the protocol (one entry for each pair).
+
+| Property                | Description                                               | Type   |
+|-------------------------|-----------------------------------------------------------|--------|
+| chain_id                | Standard chain id.                                        | number |
+| timestamp               | The timestamp of the block the pool was created on.       | number |
+| creation_block_number   | The block number this pool was created on.                | number |
+| pair_name               | The name of the pool (ie, name() in the smart contract).  | string |
+| pair_index              | The index of the token in the smart contract (one row for each pair). | number |
+| pair_address            | The contract address of the pair.                         | string |
+| pair_symbol             | The symbol of the pair.                                   | string |
+| pair_liquidity          | The liquidity available for trading for the pair (24h rolling average, in tokens).  | string |
+| pair_liquidity_usd      | The liquidity available for trading for the pair (24h rolling average, in USD).  | string |
+| volume_usd              | The volume of each pair opened and closed in the given snapshot (24h rolling average, in USD). | number |
+| open_interest_longs_usd | The sum of open interest of longs held for each pair, in USD. | number |
+| open_interest_shorts_usd | The sum of open interest in shorts held for each pair, in USD. | number |
+| liquidation_fees_usd    | The total liquidation fees accrued for each pair (24h rolling average, in USD). | number |
+| funding_rate            | The funding rate for each pair at the time of the snapshot, as a percentage. | number |
+| open_fees_usd           | The total open fees accrued for each pair (24h rolling average, in USD). | number |
+| close_fees_usd          | The total close fees accrued for each pair (24h rolling average, in USD). | number |
```

Comment:
whats the difference between this and the vault?

Reply:
here's my thinking, if we are tracking a few pairs and not all of them in the vault, we might end up underestimating the total

Reply:
vault_fees are toal fees accrued by the protocol.
those fees can only be calculated at the pair level
in the vault, it's usd values, not fees.

Reply:
ideally we want to get all token amount and usd amount as optional. in case if token price is missing we can still use our internal price table

Reply:
ok, sounds fair.